content/projects/kinoml.md

```diff
@@ -0,0 +1,37 @@
+---
+title: "KinoML"
+date: 2020-03-05T22:14:45+01:00
+nav: false
+draft: false
+# Uncomment sections below to enable them in your project
+external_resources:
+- name: KinoML repository
+  link: https://github.com/openkinome/kinoml
+  icon: github
+- name: KinoML documentation
+  link: https://openkinome.github.io/kinoml/
+  icon: book
+people:
+- key: jaime.rodriguez
+- key: talia.kimber
+- key: david.schaller
+- key: andrea.volkamer
+collaborators:
+- name: John D. Chodera
+  link: http://choderalab.org
+  more: MSKCC
+funding:
+- name: The Einstein Foundation & Stiftung Charité
+  link: https://www.einsteinfoundation.de/en/programmes/einstein-bih-visiting-fellow/
+  more: BIH Einstein Visiting Fellowship
+publications:
+# - proteinsplus2017
+---
+
+In this project, we aim to combine structure-enabled machine learning and alchemical free energy calculations to develop a predictive quantitative model to rapidly assess kinase inhibitor affinity and selectivity, design ligands with desired selectivity profiles, assess the impact of clinical point mutations on inhibitor binding and kinase activation, and determine the likelihood of different inhibitors to elicit mutational resistance.
```

Comment:
Could we make this sentence more affinity/selectivity and less mutation centric (the latter is more in the focus of miame)?
```suggestion
In this project, we aim to combine structure-enabled machine learning and alchemical free energy calculations to develop a predictive quantitative model to rapidly assess kinase inhibitor affinity and selectivity, design ligands with desired selectivity profiles and assess the impact of clinical point mutations on inhibitor binding.
```